programs/locker/src/util/token2022.rs

```diff
@@ -219,6 +222,43 @@ pub fn calculate_transfer_fee_included_amount(
     Ok(actual_amount)
 }
 
+// This function calculate the pre amount (with fee) require to transfer `amount` of token
+pub fn harvest_fees_if_available<'c: 'info, 'info>(
```

Comment:
should rename the function to `harvest_fees` because we dont check withheld_amount != 0